### DISCUSSION TITLE
Anvil Light

### DISCUSSION THREAD

**Discussion by Timofffee** [Minestom/Minestom#1553] [Help] [answered]
Hey! I tried to use the light that was in the Anvil map, but no matter how I changed the value of AmbientLight, the maps did not look identical.

Here are two screenshots. One is from the Minestom server, and the second is from the Spigot server.

How can I make the lighting the same?

![image](https://user-images.githubusercontent.com/5556081/205076350-9adebe74-3a8a-4030-abde-9137a4efbc90.png)
![image](https://user-images.githubusercontent.com/5556081/205076496-b780f1bb-18d4-434f-b5de-2ed7f1f51ac0.png)

**Comment by Obyvante** [comment, marked as answer]
Could you share your code? I do believe you have to set ambient light to **0.0f** to have anvil's lightings.

Mine is working with this code;
```java
final var dimension = DimensionType.builder(NamespaceID.from("waxeria:dimension"))
                .skylightEnabled(true)
                .ambientLight(0.0F)
                .fixedTime(1000L)
                .logicalHeight(384)
                .height(384)
                .minY(-64)
                .build();
```

**Reply by Timofffee** [reply]
Oh yeah. It works, thanks.
